### PR TITLE
feat: nextjs server-side-rendering 

### PR DESCRIPTION
**Pull request by loks0n**

### CODE REVIEW

File: nextjs/server-side-rendering/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@appwrite.io/pink": "^0.2.0",
+    "@appwrite.io/pink-icons": "^0.2.0",
+    "luke-node-appwrite-ssr": "^0.0.5",
```
Comment (felix-d1strict) [inline nextjs/server-side-rendering/package.json]: As you have switched from "luke-node-appwrite-ssr" to "node-appwrite" (https://github.com/appwrite/demos-for-react/pull/28/commits/a8d07e726317ffc338ecc9aeddfe03ff86d09326), you should ensure that the package.json is also changed.

Comment (loks0n) [reply nextjs/server-side-rendering/package.json]: We need to wait for SDK public release for this 👍

File: nextjs/server-side-rendering/.env.example
Comment (gewenyu99) [inline nextjs/server-side-rendering/.env.example:1]: Can we have top level be:

`nextjs-examples`
`react-examples`

File: nextjs/server-side-rendering/route.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { SESSION_COOKIE, createAdminClient } from "@/lib/server/appwrite";
```
Comment (gewenyu99) [inline nextjs/server-side-rendering/route.ts]: Is this a duplicate?

File: nextjs/server-side-rendering/.env.example [hunk 3]
```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_APPWRITE_ENDPOINT=
+NEXT_PUBLIC_APPWRITE_PROJECT_ID=
+APPWRITE_KEY=
```
Comment (gewenyu99) [inline nextjs/server-side-rendering/.env.example]: Wrong var name here

File: nextjs/server-side-rendering/app/oauth/route.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { SESSION_COOKIE, createAdminClient } from "@/lib/server/appwrite";
```
Comment (gewenyu99) [inline nextjs/server-side-rendering/app/oauth/route.ts]: This flow isn't working

**Review by gewenyu99 — DISMISSED** [review]

File: nextjs/server-side-rendering/.env.example
Comment (stnguyen90) [reply nextjs/server-side-rendering/.env.example:1]: ☝🏼 is a good idea. Or at the very least make the folder match the others and use the same path as the website like `example-nextjs-ssr-auth`.

Comment (loks0n) [reply nextjs/server-side-rendering/.env.example:1]: I don't see the benefit of the word example here, the whole repo is examples

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**